### DISCUSSION TITLE
[주차별 리뷰] ※20210102 ~ 20210108 1주차 과제 리뷰2※

### DISCUSSION THREAD

**Discussion by yoomambo** [4Moyede/HexatonClass-01#95] [01_주차별 리뷰]
## 지우

- 3번 : 함수 정의해서 다시 업로드

## ~~경훈~~

- ~~2번 : 함수로 구현해서 다시 업로드할 것~~
- ~~3번 : alldict, allsentence 변수 다시 바꿀 것 (노래~)~~
- ~~다른 멘티들 코드 리뷰해주기~~

## ~~재범~~

- ~~1번 :  다시 풀기~~

## 덕규

- 1번 : 멘티들꺼 참조해서 다시 해보세용~
- 2번 : function, Function은 뭔가 어떤함수인지 의미가 없지 않을까요? 다시 함수이름을 고민해보시고! 수업때 함수이름 명명에 대해서 언급했던 부분 다시 생각해보시길 바래요!
- 3번 : 다른 멘티들 과제를 차용해서 한 번 적용해보세요! 그리고 혼자서 다른 사람들의 코드를 분석해보면, 이 사람이 왜 이런 함수를 썼고, 왜 이런 변수 명, 함수 명을 썼을까? 에 대한 해답을 찾을 수 있을 겁니다!
- 다른 멘티들 코드 리뷰해주기

다시 확인하시고 pull request 보내시길 바랄게요!!
과제를 마무리 짓다보면 여러분들의 코딩 실력이 일취월장으로 늘어있을거에요
과제하느라 고생하셨고, 좀만 더 힘냅시다!

**Comment by ghost** [comment]
확인했습니다~ 1번 풀고있습니다

**Reply by jiwoo4424** [reply]
확인했습니다

**Comment by NA-DK** [comment]
확인했습니다!
